src/hotspot/share/jvmci/jvmciCompilerToVMInit.cpp

```diff
@@ -541,16 +543,17 @@ jobjectArray readConfiguration0(JNIEnv *env, JVMCI_TRAPS) {
   JVMCIObject vmFlagObj = JVMCIENV->new_VMFlag(fname, ftype, value, JVMCI_CHECK_NULL); \
   JVMCIENV->put_object_at(vmFlags, i++, vmFlagObj);                                    \
 }
-#define ADD_BOOL_FLAG(name)  ADD_FLAG(bool, name, BOXED_BOOLEAN)
-#define ADD_INT_FLAG(name)   ADD_FLAG(int, name, BOXED_LONG)
-#define ADD_INTX_FLAG(name)  ADD_FLAG(intx, name, BOXED_LONG)
-#define ADD_UINTX_FLAG(name) ADD_FLAG(uintx, name, BOXED_LONG)
+#define ADD_BOOL_FLAG(name)   ADD_FLAG(bool, name, BOXED_BOOLEAN)
+#define ADD_INT_FLAG(name)    ADD_FLAG(int, name, BOXED_LONG)
+#define ADD_SIZE_T_FLAG(name) ADD_FLAG(size_t, name, BOXED_LONG)
+#define ADD_INTX_FLAG(name)   ADD_FLAG(intx, name, BOXED_LONG)
+#define ADD_UINTX_FLAG(name)  ADD_FLAG(uintx, name, BOXED_LONG)
```

Comment:
```suggestion
#define ADD_BOOL_FLAG(name)   ADD_FLAG(bool,   name, BOXED_BOOLEAN)
#define ADD_INT_FLAG(name)    ADD_FLAG(int,    name, BOXED_LONG)
#define ADD_SIZE_T_FLAG(name) ADD_FLAG(size_t, name, BOXED_LONG)
#define ADD_INTX_FLAG(name)   ADD_FLAG(intx,   name, BOXED_LONG)
#define ADD_UINTX_FLAG(name)  ADD_FLAG(uintx,  name, BOXED_LONG)
```
Feel free to ignore, but since you are already touching this, we might as well align it.

Reply:
Leaving it as is maintains the status quo. I'm not proposing to delete the extra whitespace in front of
`ADD_FLAG`, which would be my preferred layout. And the comment does say "feel free to ignore".